.env

```diff
@@ -175,8 +175,3 @@ VOTING_BOT_SCORE_SENSITIVITY=5
 # is not easily represented.
 # This schedule is "every 15 minutes", which is close enough. For some epochs, it will run twice
 VOTING_BOT_CRON_SCHEDULE="*/15 * * * *"
-
-# ODIS Service Tests
-ODIS_SIGNER_SERVICE_URL=https://staging-pgpnp-signer0.azurefd.net
```

Comment:
We should keep these here so that Vscode debugging works. We can have a copy in both locations if needed.

Reply:
👍